### ISSUE TITLE
Having issues with proxy tags

### ISSUE THREAD

**Issue by ghost** [almonds0166/pk.py#24]
Hi. When i try [the example in the docs](https://pluralkit.readthedocs.io/en/latest/source/v2/api.html#pluralkit.ProxyTag), I get this error:
```py
Traceback (most recent call last):
  File "(file location)", line 3, in <module>
    pt = ProxyTag("{", "}")
TypeError: ProxyTag.__init__() missing 1 required keyword-only argument: 'proxy_tag'
```
For that all I'm trying is:
```py
from pluralkit import ProxyTag

pt = ProxyTag("{", "}")
pt("This is an example.")
```

Is there something I'm missing, or is this a bug? I'm having similar issues with trying to set proxy tags.

**Comment by mimuki** [comment]
(ack, the perils of pressing enter too soon- please hold on a sec)

**Comment by almonds0166** [comment]
Thanks for finding this and bringing it up! The bug is that an argument (`proxy_tag`) that should be optional was not made optional in the code. I'll fix this and let you know.

**Comment by almonds0166** [comment]
Should be fixed now in v1.1.5 :slightly_smiling_face: 

```bash
pip install -U pluralkit
```
